app/services/slack/messengers/article_published.rb

```diff
@@ -23,9 +23,10 @@ def call
           url: URL.article(article),
         )
 
+        # [forem-fix] Remove channel name from SiteConfig
         Slack::Messengers::Worker.perform_async(
           message: message,
-          channel: "activity",
+          channel: SiteConfig.article_published_slack_channel,
```

Comment:
I think we might want a guard clause here to prevent the worker from running if this variable is blank.

Reply:
There is a `default: "activity"`, wouldn't that be enough?

Reply:
Oh, good point 😅